config/vufind/EDS.ini

```
@@ -504,3 +504,18 @@ DetailPageFormat = 'Long'
 ; Default is false.
 ; Note: Customers with custom catalog databases may need to leave this disabled.
 ;ContentProvider = false
+
+[Catalog]
+; This section enables ILS Driver for records from your catalog database in EDS.
+; set EDSHasCatalog to true and ensure your ILS Driver has been configured
+EDSHasCatalog = false
+; Please enter the Database ID for your custom catalog database, it typically has the format cat12345a
+CatalogDatabaseId = ""
+; Provide a list of regular expressions and their replacement value in the next two parameters
+; Please note: Kindly ensure that EBSCO indexes your Catalog Record Number (or RTAC-ID) as the Accession Number
+; Typically EBSCO will add a prefix to your the Catalog Record Number, e.g. abc.1234
+; For OAI-PMH Catalogs, you will often find a long prefix and a change where hyphens are replaced with periods
+; CatalogANRegex[] = "/^abc\./i"
+; CatalogANRegex[] = "/\./"
+; CatalogANReplace[] = ""
+; CatalogANReplace[] = "-"
```

Maybe this would be more readable if you paired up the matching parts:

```suggestion
; CatalogANRegex[] = "/^abc\./i"
; CatalogANReplace[] = ""
; CatalogANRegex[] = "/\./"
; CatalogANReplace[] = "-"
```

---

Now that you point it out it is obviously way better :) Sorry I didn't think about that myself